python/ray/data/dataset.py

```diff
@@ -2788,6 +2789,65 @@ def write_json(
             **pandas_json_args,
         )
 
+    @PublicAPI(stability="alpha")
+    @ConsumptionAPI
+    def write_images(
+        self,
+        path: str,
+        column: str,
```

Comment:
why we require users to set `column` by default? Should this be optional? By default we should assume the dataset should only have one column, and users want to write it as an image, right?